go.mod

```diff
@@ -28,3 +28,5 @@ require (
 replace github.com/jteeuwen/go-bindata => github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
 
 replace vbom.ml/util => github.com/fvbommel/util v0.0.0-20180919145318-efcd4e0f9787
+
+replace github.com/openshift/library-go => /Users/lszaszki/go/src/github.com/openshift/library-go
```

Comment:
Just a reminder to remove this once https://github.com/openshift/library-go/pull/1020 merges

Reply:
/hold
for this, release when ready :)

Reply:
done

/hold cancel